src/commands/ext-uninstall.ts

```diff
@@ -29,72 +30,82 @@ export default new Command("ext:uninstall <extensionInstanceId>")
       }
       throw err;
     }
-    let confirmedServiceAccountDeletion;
     if (!options.force) {
+      const serviceAccountMessage = `Uninstalling deletes the service account used by this extension instance:\n${clc.bold(
+        instance.serviceAccountEmail
+      )}\n\n`;
       const resourcesMessage = _.get(instance, "config.source.spec.resources", []).length
-        ? "This will delete the following resources \n" +
+        ? "Uninstalling deletes all extension resources created for this extension instance:\n" +
           instance.config.source.spec.resources
-            .map((resource: extensionsApi.Resource) => `- ${resource.type}: ${resource.name} \n`)
-            .join("")
+            .map((resource: extensionsApi.Resource) =>
+              clc.bold(`- ${resource.type}: ${resource.name} \n`)
+            )
+            .join("") +
+          "\n"
         : "";
-      const extensionDeletionMessage = `You are about to uninstall extension ${clc.bold(
-        instanceId
-      )} from project ${clc.bold(projectId)}.\n${resourcesMessage}Are you sure?`;
+      const artifactsMessage =
+        `The following ${clc.bold("will not")} be deleted:\n` +
+        "Any artifacts (for example, stored images) created by this extension instance.\n" +
+        "Any other project resources with which this extension instance interacted.\n";
+
+      const extensionDeletionMessage =
+        `Here's what will happen when you uninstall ${clc.bold(instanceId)} from project ${clc.bold(
+          projectId
+        )}. Be aware that this cannot be undone.\n\n` +
+        `${serviceAccountMessage}` +
+        `${resourcesMessage}` +
+        `${artifactsMessage}`;
+
+      logger.info(extensionDeletionMessage);
       const confirmedExtensionDeletion = await promptOnce({
         type: "confirm",
         default: true,
-        message: extensionDeletionMessage,
+        message: "Are you sure?",
```

Comment:
just realized that there's a lot of text b/w the command and the this confirmation.  Could we have this be:

Are you sure that you want to uninstall this extension?

Reply:
Done!